docker/env/build_n_push.sh

```diff
@@ -0,0 +1,32 @@
+# !/usr/bin/env bash
+set -e
+
+DOCKER_ENV_DIR=$(dirname $(readlink -f $0 ))
+VERSION=v$(cat ${DOCKER_ENV_DIR}/version)
+
+SCT_DIR=$(dirname $(dirname ${DOCKER_ENV_DIR}))
+PY_PREREQS_FILE=requirements-python.txt
+CENTOS_PREREQS_FILE=install-prereqs.sh
+
+if [[ ! -z "`docker images scylladb/hydra:${VERSION} -q`" ]]; then
+    echo "Image up-to-date"
+else
+    echo "Image with version $VERSION not found. Building..."
+    cd ${DOCKER_ENV_DIR}
+    cp -f ${SCT_DIR}/${PY_PREREQS_FILE} .
+    cp -f ${SCT_DIR}/${CENTOS_PREREQS_FILE} .
+    docker build --network=host -t scylladb/hydra:${VERSION} .
+    rm -f ${PY_PREREQS_FILE} ${CENTOS_PREREQS_FILE}
+    cd -
+fi
+
+echo "Tagging and pushing"
+docker push scylladb/hydra:${VERSION}
+
+regions=( eu-west-1 us-east-1 )
+for region in  "${regions[@]}"
+do
+    DOCKER_REPO=$(aws ecr describe-repositories --region ${region} --repository-names scylladb/hydra |  python -c "import sys, json; print(json.load(sys.stdin)['repositories'][0]['repositoryUri'])")
```

Comment:
I shouldn't be reading the docs.. doing `aws ecr describe-repositories` doesn't show `--query`

Reply:
fixed

Reply:
> I shouldn't be reading the docs.. doing `aws ecr describe-repositories` doesn't show `--query`

Yep. Because it's a common options. Described in `$ aws help`